go.mod

```diff
@@ -1,5 +1,8 @@
-module github.com/in-toto/in-toto-golang
+module github.com/shibumi/in-toto-golang
```

Comment:
@SantiagoTorres  oops.. we might want to fix this. No idea how that change got in my commit :D 
I will compile a PR for this tomorrow (too late in Germany already).

Same for the go version